src/util/border-style.js

```diff
@@ -0,0 +1,24 @@
+
+import assign from 'object-assign'
+
+/**
+ * Utility for extracting border style props from components
+ */
+
+const sides = ['Top', 'Right', 'Bottom', 'Left']
+
+function borderStyle (props) {
```

Comment:
Perhaps the function should be named something else since `borderStyle` is a style property.